### PR TITLE
Manual updates 20220215 split GPS

### PR DESCRIPTION
**Pull request by moljac**
### Does this change any of the generated binding API's?

Yes. Updates, new dependencies, etc

### Describe your contribution

PR split

https://github.com/xamarin/GooglePlayServicesComponents/pull/575

### CODE REVIEW

**Comment by moljac** [comment]
Nuget restore error:

```


./generated/com.google.android.gms.play-services-ads/com.google.android.gms.play-services-ads.csproj : error NU1605: Detected package downgrade: Xamarin.AndroidX.Core from 1.7.0.2 to 1.3.2.3. Reference the package directly from the project to select a different version.  [./generated/GooglePlayServices.sln]
./generated/com.google.android.gms.play-services-ads/com.google.android.gms.play-services-ads.csproj : error NU1605:  Xamarin.GooglePlayServices.Ads -> Xamarin.GooglePlayServices.Ads.Lite -> Xamarin.AndroidX.Work.Runtime 2.7.1.3 -> Xamarin.AndroidX.Core (>= 1.7.0.2)  [./generated/GooglePlayServices.sln]
./generated/com.google.android.gms.play-services-ads/com.google.android.gms.play-services-ads.csproj : error NU1605:  Xamarin.GooglePlayServices.Ads -> Xamarin.AndroidX.Core (>= 1.3.2.3) [./generated/GooglePlayServices.sln]
./generated/com.google.firebase.firebase-ads/com.google.firebase.firebase-ads.csproj : error NU1605: Detected package downgrade: Xamarin.AndroidX.Core from 1.7.0.2 to 1.3.2.3. Reference the package directly from the project to select a different version.  [./generated/GooglePlayServices.sln]
./generated/com.google.firebase.firebase-ads/com.google.firebase.firebase-ads.csproj : error NU1605:  Xamarin.Firebase.Ads -> Xamarin.GooglePlayServices.Ads -> Xamarin.GooglePlayServices.Ads.Lite -> Xamarin.AndroidX.Work.Runtime 2.7.1.3 -> Xamarin.AndroidX.Core (>= 1.7.0.2)  [./generated/GooglePlayServices.sln]
./generated/com.google.firebase.firebase-ads/com.google.firebase.firebase-ads.csproj : error NU1605:  Xamarin.Firebase.Ads -> Xamarin.GooglePlayServices.Ads -> Xamarin.AndroidX.Core (>= 1.3.2.3) [./generated/GooglePlayServices.sln] 
```

**Comment by moljac** [comment]
GPS depending on MLKit:

```
Dependency errors : 1
1
	System.Exception: 
No matching artifact config found for: 
			com.google.mlkit.barcode-scanning-common:16.0.0
to satisfy dependency of: 
			com.google.android.gms.play-services-mlkit-barcode-scanning:17.0.0

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.mlkit",
        "artifactId": "barcode-scanning-common",
        "version": "16.0.0",
        "nugetVersion": "CHECK PREFIX 16.0.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
```

File: samples/com.google.android.gms/play-services-fitness/BasicSensorsApi/MainActivity.cs [hunk 1]
```diff
@@ -74,7 +74,7 @@ void buildFitnessClient()
             // Create the Google API Client
             mClient = new Android.Gms.Common.Apis.GoogleApiClient.Builder (this)
                 .AddApi(Android.Gms.Fitness.FitnessClass.SENSORS_API)
-                .AddScope(new Android.Gms.Common.Apis.Scope(Android.Gms.Common.Scopes.FitnessLocationRead))
+                //.AddScope(new Android.Gms.Common.Apis.Scope(Android.Gms.Common.Scopes.FitnessLocationRead))
```
Comment (jonpryor) [inline samples/com.google.android.gms/play-services-fitness/BasicSensorsApi/MainActivity.cs:77]: Is this removal needed because of an API change/break?

Why comment out and not just remove?
